src/tests/compiler/tools/languageServiceTests.ts

```diff
@@ -112,6 +112,57 @@ describe(nameof(LanguageService), () => {
             });
         });
     });
+
+    describe(nameof<LanguageService>(l => l.getEditsForRefactor), () => {
+        it("should get edits for known refactor 'move to a new file'", () => {
+            const { sourceFile, project } = getInfoFromText("export class A {}; function f(){return new A(); }", { filePath: "/file.ts" });
+            const classId = sourceFile.getClassOrThrow("A").getNameNodeOrThrow();
+            const results = project.getLanguageService().getEditsForRefactor(sourceFile.getFilePath(), {},
+                { pos: classId.getStart(), end: classId.getEnd() }, "Move to a new file", "Move to a new file", {});
+
+            const edit1 = results!.getEdits().find(edit => edit.getFilePath() === sourceFile.getFilePath());
+            const edit2 = results!.getEdits().find(edit => edit.getFilePath() === "/A.ts");
```

Comment:
Does edits return more than two results? Could you add an assertion for how many results it should return?